src/flame-chart.ts

```diff
@@ -90,13 +97,21 @@ export class FlameChart extends FlameChartContainer<FlameChartStyles> {
             flameChartPlugin.on('select', (node, type) => this.emit('select', node, type));
 
             if (waterfall) {
-                activePlugins.push(new TogglePlugin(flameChartName, { styles: styles?.togglePlugin }));
+                activePlugins.push(
+                    new TogglePlugin({ title: flameChartName, settings: { styles: styles?.togglePlugin } })
+                );
             }
 
             activePlugins.push(flameChartPlugin);
             activePlugins.unshift(timeframeSelectorPlugin);
         }
 
+        if (timeseries) {
+            timeseries.forEach((ts, idx) => {
```

Comment:
I think this behavior is not expected for our users. Still, it is better to provide for the possibility of adding several charts to one plugin by default. If a user needs multiple plugins, then it is better to use a flame chart container. Otherwise, it will be very confusing, as it seems to me. @Smrtnyk, what do you think?

Reply:
Obviously you know this project best - but the public api that you defined does not expose the internals of the plugins - A user can provide mulitple time-series if they wish...

Materially - It is another chunk of work to refactor the to do this - and i have not planned to do this - as the plugin already meets my use-cases

I would love to chat to you about how i am using it - We have built a custom sampling profiler and associated ui- but the killer feature is the ability to compare multiple profiles with (cpu, memory, other custom attributes too) this has worked almost completly out of the box - this project of your is very very cool!

Reply:
I agree with Nikolay here

Reply:
It's very cool that you liked the project.)

I think in terms of user experience for the purpose of displaying only one flame chart. And in this case, several graphs at the very bottom after the flame chart look rather strange. For example, in chrome devtools, all charts are built right into the timeframe selector. In our case, of course, we can expand this story and make a separate graph. But at the same time, it should not take up too much space - that's why I suggested making it possible to render several charts at once. After the merge, I'm basically going to slightly improve the api of the plugin that you suggest.

Your goals are most likely related to using the `FlameChartContainer`, which allows you to create complex flame charts. And there you can use the plugins however you like.)

And the `FlameChart` class should, in theory, only allow one chart and its associated information to be displayed.)

Well, for the rest - you can write to me in the telegram. I am available there under the same name.)